### PR TITLE
Updates v1.28 hugo.toml for release v1.32

### PR DESCRIPTION
**Pull request by chanieljdan**
Updating hugo.toml for release v1.32.

/hold until v1.32 release day.

### CODE REVIEW

**Comment by reylejano** [comment]
Retried the Deploy preview on Netlify and it was a success
https://deploy-preview-48986--k8s-v1-28.netlify.app/

File: hugo.toml [hunk 1]
```diff
@@ -184,35 +184,35 @@ js = [
...
-version = "v1.31"
-githubbranch = "v1.31.0"
+version = "v1.32"
+githubbranch = "v1.32.0"
 docsbranch = "main"
 url = "https://kubernetes.io"
 
+[[params.versions]]
+version = "v1.31"
+githubbranch = "v1.31.4"
```
[the remark this answers is not in the document]
Comment (reylejano) [reply hugo.toml:194]: Same for the following patch versions on this PR, I only see the -1 patch version on releases.kubernetes.io

[the remark this answers is not in the document]
Comment (chanieljdan) [reply hugo.toml:194]: Thanks Rey, release date will be Dec 11, which is 1 day after the scheduled patch release date. Should I just use what's currently the latest patch or what should be the latest patch by Wed?

[the remark this answers is not in the document]
Comment (reylejano) [reply hugo.toml:194]: Yes, use what's currently the latest patch

[the remark this answers is not in the document]
Comment (chanieljdan) [reply hugo.toml:194]: Sounds good, will update the patch versions

**Pull request merged**